CMakeLists.txt

```diff
@@ -87,6 +120,7 @@ set(libqmatrixclient_SRCS
     lib/networksettings.cpp
     lib/converters.cpp
     lib/util.cpp
+    lib/encryptionmanager.cpp
```

Comment:
Just noticed - the respective lines (and additional lines for header files) should be added to `libqmatrixclient.pri` as well to support building with `qmake`.